include/wx/graphics.h

```diff
@@ -34,7 +34,7 @@ enum wxInterpolationQuality
     // default interpolation
     wxINTERPOLATION_DEFAULT,
     // no interpolation
-    wxINTERPOLATION_NONE, 
+    wxINTERPOLATION_NONE,
```

Comment:
It would be nice to leave out the whitespace-only changes or make them in a separate commit if possible (here and elsewhere).